packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -23,7 +23,9 @@ const printCategory = ({label, key}: {label: string; key: number}) => {
 
 const printVersions = ({version, versions, versionRange}) => {
   if (versions) {
-    const versionsToShow = versions.join(', ');
+    const versionsToShow = Array.isArray(versions)
+      ? versions.join(', ')
+      : 'N/A';
```

Comment:
Please update the `HealthCheckResult` types to account for this type, so we can avoid similar issues in the future

Reply:
Hey @thymikee, thanks very much for the pointers, I have made the changes :) Thanks!